project-chip/connectedhomeip
# Changed the bridge-app to use temperature sensors rather than switches.

#### Problem
Updated the bridge-app to use temperature sensors rather than switches to match recent changes to the bridge test plan.

Running some of the commands for the bridge test plan and action clusters test plan caused a "Chip stack locking error".

#### Change overview
The switches in the bridge-app were change to temperature sensors to match
the updated bridge test plan.

A lock was also added in a few places in the bridge-app to fix errors that reported reported
a "Chip stack locking error"

#### Testing
1. Start bridge app and wait for initialization to complete
2. Run chip-tool pairing
    chip-tool pairing ethernet 12344321 20202021 3840 localhost 5540
3. Get the measured-value for a temperature sensor
   chip-tool temperaturemeasurement read measured-value 12344321 4
4. Press t in the bridge-app to change the measured-value for all temperature sensors
5. Get the measured-value for the same temperature sensor and verify that it has changed.
